lib/split-home-site-urls/asset-urls.php

```diff
@@ -0,0 +1,309 @@
+<?php
+/*
+ Plugin name: Rewrite Static URLs
+ Description: Rewrite static assets to be served from a different URL
+ Version: 1.5
+ Author: Erick Hitter
+ Author URI: https://ethitter.com/
+ License: GPLv2
+*/
+
+/*
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; version 2 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+namespace Automattic\VIP\Split_Home_Site_URLs;
+
+class Asset_URLs {
+	/**
+	 * Singleton
+	 */
+	private static $instance = null;
+
+	/**
+	 * Class variables
+	 */
+	private $static_host = null;
+
+	/**
+	 * Silence is golden!
+	 */
+	private function __construct() {}
+
+	/**
+	 * Instantiate singleton
+	 */
+	public static function get_instance() {
+		if ( ! is_a( self::$instance, __CLASS__ ) ) {
+			self::$instance = new self;
```

Comment:
Probably should use empty parens `()` here - I thought that was part of the WP standards but I didn't actually see it.